### PR TITLE
[KOA-5473] Update components to semantic tokens

### PR DESCRIPTION
**Pull request by anambl**
|Before|After|
|-|-|
|image|image|
|image|image|
|image|image|
|image|image|

Remember to include the following changes:

- [x] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack/blob/main/CHANGELOG_FORMAT.md))
- [ ] `README.md`
- [ ] Tests
- [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: .storybook/themeableAttributes.js [hunk 1]
```diff
@@ -18,7 +18,6 @@
 
 /* @flow */
 
-import { themeAttributes as accordionThemeAttributes } from '../packages/bpk-component-accordion';
```
Comment (frugoman) [inline .storybook/themeableAttributes.js:21]: is this change related to update semantic tokens?

File: packages/bpk-component-accordion/index.js [hunk 2]
```diff
@@ -22,12 +22,10 @@ import BpkAccordion from './src/BpkAccordion';
...
 import withSingleItemAccordionState from './src/withSingleItemAccordionState';
 import withAccordionItemState from './src/withAccordionItemState';
-import themeAttributes from './src/themeAttributes';
 
 export {
   BpkAccordion,
   BpkAccordionItem,
   withSingleItemAccordionState,
   withAccordionItemState,
-  themeAttributes,
```
Comment (frugoman) [inline packages/bpk-component-accordion/index.js:32]: same as above. is this needed/related?

File: packages/bpk-component-accordion/src/BpkAccordionItem.module.scss [hunk 3]
```diff
@@ -84,27 +63,7 @@ $bpk-spacing-v2: true;
...
-        --bpk-accordion-hover-color,
-        $bpk-color-sky-blue-shade-03
-      );
-    }
-
-    &:active {
-      @include bpk-themeable-property(
-        fill,
-        --bpk-accordion-active-color,
-        $bpk-color-sky-blue-shade-03
```
Comment (frugoman) [inline packages/bpk-component-accordion/src/BpkAccordionItem.module.scss:105]: do we need to also remove this vars where they were declared? (maybe not, just to know)

File: .storybook/themeableAttributes.js (hunk 1 above)
Comment (anambl) [reply .storybook/themeableAttributes.js:21]: Not exactly. But we use these theming for `hover` and `active` states which we don't need in the accordion component as they will all have the same colours. So we can just get rid of them. It's more like cleaning up the code as we go

File: packages/bpk-component-accordion/index.js (hunk 2 above)
Comment (anambl) [reply packages/bpk-component-accordion/index.js:32]: yep same as above, just cleaning up as we don't need this anymore

File: packages/bpk-component-accordion/src/BpkAccordionItem.module.scss (hunk 3 above)
Comment (anambl) [reply packages/bpk-component-accordion/src/BpkAccordionItem.module.scss:105]: same 😬

**Review by frugoman — APPROVED** [review]
